bson/binary.py

```diff
@@ -183,6 +183,13 @@ class UuidRepresentation:
 .. versionadded:: 4.0
 """
 
+SENSITIVE_SUBTYPE = 8
+"""BSON binary subtype for sensitive data.
```

Comment:
Are users ever expected to use this?

Reply:
The spec doesn't mention it, but other drivers expose these constants, e.g. https://github.com/mongodb/mongo-go-driver/blob/4acd55139b846e6bfdfe316770a5b1b646d5ff96/bson/types.go#L47.